src/js/controller.js

```diff
@@ -470,13 +470,55 @@ export const MediaUIStates = {
       return getShowingSubtitleTracks(controller).map(({ kind, label, language }) => ({ kind, label, language }));
     },
     mediaEvents: ['loadstart'],
-    trackListEvents: ['addtrack', 'removetrack', 'change'],
+    textTracksEvents: ['addtrack', 'removetrack', 'change'],
+  },
+  MEDIA_RENDITION_LIST: {
+    get: function (controller) {
+      const { media } = controller;
+      const selected = media?.videoTracks?.[media.videoTracks?.selectedIndex ?? 0];
+
+      if (!selected) return [];
+
+      return [...selected.renditions ?? []];
+    },
+    mediaEvents: ['loadstart'],
+    renditionListEvents: ['addrendition', 'removerendition'],
+  },
+  MEDIA_RENDITION_ENABLED: {
```

Comment:
good call, will change to "renditions".

The rendition API supports enabling just a few renditions. Maybe only 270p and 360p for whatever reason. 
This would allow that. 

It's also one of the reasons why the `selectedIndex` property for renditions in media ui ext doesn't make sense.